presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,68 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+
+* Fix a correctness bug for queries with a ``LIMIT`` on top of a ``FULL OUTER JOIN``.
+* Fix a correctness bug in how word boundaries are handled for regular expression functions when using the Joni regex engine.
+* Fix query failures caused by Parquet predicate pushdown for smallint and tinyint types.
+* Fix query failures when view columns were fully qualified with catalog and/or schema.
+* Fix query failures when creating and reading an unpartitioned Hive table within the same transaction.
+* Fix an issue that would disable a performance optimization if two tables with mismatching
+  but compatible bucket counts are joined and one of the table has a `$bucket` filter.
+* Fix an issue that may cause procedure calls to fail if no queries were run after the server started up.
+* Fix an issue where the analyze properties would remain in the ``system.metadata.analyze_properties`` table even after the connector was removed.
+* Add :func:`ST_Length` for spherical geography.
+* Add ``view_owner`` column to the ``information_schema.views`` system table.
+* Add a ``JSON`` version of the query plan to ``QueryCompletedEvent``.
+* Add support for creating warnings during parsing.
+* Add a warning for using the ``current_role`` reserved word as an identifier.
+* Add support for using the empty string as a delimiter for the :func:`split` function.
+  When an empty string is used as the delimiter, the string will be split into individual characters.
+
+Raptor Changes
+--------------
+* Add ``raptor.minimum-node-count`` configuration property. The the number of nodes in a cluster is fewer
+  than the configured amount, data recovery and reassignment will not run.
+* Add configuration property ``raptor.startup-grace-period`` to control the delay of the initial bucket balancer run after the coordinator startup.
+
+Hive Connector Changes
+----------------------
+
+* Add ``hive.ignore-table-bucketing`` configuration property and ``ignore_table_bucketing`` session property.
+  When set to true, these properties enable reading from partitions whose bucketing scheme does not match the table bucketing scheme.
+
+Verifier Changes
+----------------
+* Fix an issue where the determinism check for ``INSERT`` queries were not run when there was a result mismatch.
+* Fix an issue where the checksum query was treated as the main query, and the main query was treated as part of the setup queries,
+  for ``INSERT`` query verification.
+* Fix query timeout enforcement by replacing local timer with session property ``query_max_execution_time``.
+* Improve result comparison for floating point columns by using relative errors.
+  Replace configuration property ``expected-double-precision`` with ``relative-error-margin``.
```

Comment:
Is replace a rename? If so let's say `rename` as we do below for many other configs.

Reply:
It's not a rename. `expected-double-precision` is the number of decimal points that needs to be matched, while `relative-error-margin` is the max tolerable relative error.